package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "teams-for-linux",
-  "version": "1.3.26",
+  "version": "1.3.27",
```

```suggestion
  "version": "1.4.0",
```

I think is worth starting upping the second number when updating electron to a mayor version, as we had a few problems and remembering 1.3.xx is a bit of a pain.